homepage/models.py

```diff
@@ -40,25 +44,37 @@ class Req_Code(models.IntegerChoices):
     req_code = models.SmallIntegerField(choices=Req_Code.choices, default=Req_Code.NONE)
 
 
-class User(models.Model):
-    name = models.CharField(max_length=100)
-    email = models.CharField(max_length=50)
-    password = models.CharField(max_length=30)
+class AppUser(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+
+    def __str__(self):
+        return self.user.username
 
 
 class Review(models.Model):
     course = models.ForeignKey(Course, on_delete=models.CASCADE)
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    user = models.ForeignKey(AppUser, on_delete=models.CASCADE)
     rate = models.SmallIntegerField()
     date = models.DateTimeField(default=timezone.now)
     content = models.TextField()
     course_load = models.SmallIntegerField()
 
+    def __str__(self):
+        MAX_WORDS_PREVIEW = 5
+        shortened_review = ' '.join(self.content.split()[:MAX_WORDS_PREVIEW])
+        return 'Preview: {}'.format(shortened_review)

```

Comment:
with python3 we have a great format style like:
```suggestion
        return f'Preview: {shortened_review}'

```

Reply:
Changed it